GameData/RealismOverhaul/RO_SuggestedMods/ModularLaunchPads/MLP.cfg

```diff
@@ -1,5 +1,13 @@
-@PART[AM_MLP*]:FOR[RealismOverhaul]
+@PART[AM_MLP]
 {
-    %rescaleFactor = 1.61
-    %RSSROConfig = True
+        %rescaleFactor = 1.61
+        RSSROConfig = True
+}
+
+@PART[AM_MLP]:FOR[RP-0]
+{
+    %TechRequired = start
+    %cost = 5
+    %entryCost = 5
+    RP0conf = true
 }
```

Comment:
Please PR the RP-1 configuration portion into RP-1, not into RO core.  We'll merge that bit there.
Actually, please revert the changes to this file.  Removing the pass specifier isn't helpful, not sure what all parts are included in MLP that the * on the PART should also be removed.  (The current patch matches what I took from MLP's repo history.)

Reply:
No problem.

Reply:
I've reverted the config to be the original now.